project/Dependencies.scala

```diff
@@ -7,19 +7,18 @@ object Dependencies {
     val aws = "2.21.41"
     val jackson = "2.15.2"
     val awsRds = "1.12.629"
-    val enumeratumPlay = "1.7.3"
+    val enumeratumPlay = "1.8.0"
   }
 
   // https://github.com/orgs/playframework/discussions/11222
+  // We no longer have any vulnerabilities through Jackson, but still need to defined jackson dependencies.
+  // Jackson does not like having different versions of its packages installed.
   private val jacksonOverrides = Seq(
     "com.fasterxml.jackson.core" % "jackson-core",
     "com.fasterxml.jackson.core" % "jackson-annotations",
+    "com.fasterxml.jackson.core" % "jackson-databind",
     "com.fasterxml.jackson.datatype" % "jackson-datatype-jdk8",
     "com.fasterxml.jackson.datatype" % "jackson-datatype-jsr310",
-    "com.fasterxml.jackson.core" % "jackson-databind"
-  ).map(_ % Versions.jackson)
-
-  private val akkaSerializationJacksonOverrides = Seq(
```

Comment:
Was this rolled into the main jackson overrides for any particular reason? (Other than pekko replacing akka here)

Reply:
Pekko still uses Jackson, so I guess I could have just renamed it to `pekkoSerializationJacksonOverrides` but to be honest I didn't see too much of a reason to keep them separate anymore. Other dependencies like the AWS SDK also import Jackson and we don't explicitly have a section for that, so why have one specifically for Pekko.